pkgs/tools/text/jsawk/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, makeWrapper, spidermonkey }:
+
+stdenv.mkDerivation rec {
+  name = "jsawk-${version}";
+  version = "1.5-pre";
+  src = fetchFromGitHub {
+    owner = "micha";
+    repo = "jsawk";
+    rev = "5a14c4af3c7b59807701b70a954ecefc6f77e978";
+    sha256 = "0z3vdr3c8nvdrrxkjv9b4xg47mdb2hsknxpimw6shgwbigihapyr";
+  };
+  dontBuild = true;
+  buildInputs = [ makeWrapper ];
+  installPhase = ''
+    mkdir -p $out/bin
+    cp $src/jsawk $out/bin/
+    wrapProgram $out/bin/jsawk \
+      --prefix PATH : "${spidermonkey}/bin"
+  '';
+
+  meta = {
+    description = "Jsawk is like awk, but for JSON";
+    homepage = https://github.com/micha/jsawk;
+    license = "unknown";
```

Comment:
I can't find a license specified. What should the value be here?

Reply:
If there's no license, it means that by default it should be `unfree`. But there is, https://github.com/micha/jsawk/blob/5a14c4af3c7b59807701b70a954ecefc6f77e978/jsawk#L11 says "no rights reserved", so I feel like it can be `publicDomain`? But then, the PKGBUILD says "unknown", and the author accepted it too: https://github.com/micha/jsawk/pull/24, so it's slightly conflicting there. But I still think it's ok to go with `publicDomain`. Otherwise maybe you'd want to ask upstream for clarification about "no rights reserved" vs "unknown"?

Reply:
@vyp I think it's safe to call it public domain based on the "no rights reserved"

Licenses :crying_cat_face: